tasties_app/tests/test_recipes_view.py

```diff
@@ -34,6 +36,20 @@ def test_view_recipe_not_found(self, client, signed_up_credentials, login_to_sit
         assert response.status_code == 302
         assert response.url == '/'
 
+    def test_add_comment_function(self, client, comment, recipe_test, create_user):
+        client.force_login(create_user)
+        response = client.get(f'/view_recipe/{recipe_test.id}/?comment-adding={comment.comment_text}')
+        assert response.status_code == 200
+        comment_test = Comment.objects.get(author_id=comment.author_id, comment_text=comment.comment_text)
+        assert comment_test.comment_text == comment.comment_text
+
+    def test_add_comment_function_invalid(self, client, create_user, recipe_test, comment):
+        client.force_login(create_user)
+        response = client.get(f'/view_recipe/{recipe_test.id}/?comment-adding={""}')
+        assert response.status_code == 200
+        with pytest.raises(ObjectDoesNotExist):
+            Comment.objects.get(author_id=comment.author_id, comment_text="")
```

Comment:
This statement `Comment.objects.get(author_id=comment.author_id, comment_text="")` can potentially return a `Comment`, worse - it can return multiple `Comments`. I see you're trying to `.get` a comment with no text - this kind of stuff can be forced into the database by an `admin`, for example - even though it's an invalid input.

Reply:
You're right, but since you can't create a comment with an empty string, I just create it in the line above (line 48) and see that it's not saved.
I try to do that with a user not a admin so it is ok.